src/mementoappserver.cpp

```diff
@@ -190,9 +191,11 @@ void MementoAppServerTsx::on_initial_request(pjsip_msg* req)
     served_user = uri_to_string(PJSIP_URI_IN_ROUTING_HDR, uri);
 
     pjsip_param* sescase = pjsip_param_find(&psu_hdr->other_param, &SESCASE);
+    pjsip_param* orig_cdiv = pjsip_param_find(&psu_hdr->other_param, &ORIG_CDIV);
```

Comment:
Worth a comment to the effect that we treat `orig-cdiv` as originating (which means the subscriber will see the call in their call logs).  We generally recommend that customers don't put memento in their `orig-cdiv` IFCs for this very reason.